python/semantic_kernel/orchestration/sk_context.py

```diff
@@ -5,6 +5,9 @@
 
 import pydantic as pdt
 
+from semantic_kernel.connectors.ai.open_ai.models.chat.function_call import (
+    FunctionCall,
+)
```

Comment:
Would really like to avoid having `SKContext` take a dependency on the OpenAI connector. One option would be to move the `FunctionCall` model elsewhere --  although we've previously spoken about trying to keep function calling code in the connector as much as possible. Alternatively, is there another way to track the function call without using `SKContext`?

Reply:
now created a objects dict in context that can be used to hold anything other then a string (since we use variables for those), and adapted the other code to that model.